src/Components/Contributors/ContributorModal.tsx

```diff
@@ -108,6 +109,13 @@ async function searchCristinPersons(authors: ImportPublicationPerson[]) {
         url: CRIST_REST_API + '/persons/' + cristinPerson.cristin_person_id + '?lang=' + SearchLanguage.En,
         order: i + 1,
         identified_cristin_person: cristinPerson.identified_cristin_person,
+        require_higher_authorization: cristinPerson.require_higher_authorization,
+        badge_type: getBadgeForContributor(
+          affiliations,
+          cristinPerson.require_higher_authorization,
```

Comment:
litt bedre for lesbarheten å splitte paramatre inni funksjonen : getBadgeForContributor(affiliations, cristinPerson)